content/about/historiek.md

```diff
@@ -83,18 +83,25 @@ De voorbije jaren zwaaiden veel van de oudere zeus-leden af, en moesten ze ons m
 
 De Zeus draait nog steeds verder, niet elk jaar even sterk maar nog steeds met veel enthousiasme.
 
-# De jaren 2010-2016
+# De jaren 2010-2014
 
 Je zou het bijna een 2de geboorte kunnen noemen, Zeus ontploft. Wat onze hoeveelheid code betreft alvast.
 Er spruiten projecten uit de grond, veel projecten. Extern, zoals FK-Enrolment, Gandalf, 12Urenloop, en Hydra.
-Maar ook intern, zoals Tab/p, Slotmachien, Haldis, en Gamification. We automatiseren alles!
 Veel van deze projecten verdienen hun eigen beschrijving, want ze zijn stuk voor stuk symbool voor de innovatie en creatiedrang die Zeus doet ontsproeien in studenten, je kan daarom een compleet overzicht zien op onze [projectpagina](https://zeus.ugent.be/projects/).
 
+# Het jaar 2014-2015
+
+De eerste stapjes van de kelderautomatisering worden gezet tijdens de zomervakantie, Slotmachien wordt tot leven gewekt. Vertrouwde leden kunnen nu zonder sleutel binnen en buiten in de kelder. Het academiejaar gaat van start met de (ondertussen meermaals terugkerende en steevast enorm populaire) rondleiding van het Datacentrum. Het High Performance Computing team van DICT laat ons kennismaken met de snelste computer van het land (op dat moment (snif)). Er worden weer lustig lessen en workshops georganiseerd, en voor de afwisseling wordt de eerste Zeus Spelletjesavond gehouden.
```

Comment:
s/(snif)/*snif*/

Reply:
> en voor de afwisseling wordt de eerste Zeus Spelletjesavond gehouden.

https://zeus.ugent.be/blog/07-08/spellekens-avond/